test/test_filter_masking.rb

```diff
@@ -93,5 +99,38 @@ def filter(config, messages)
       filtered_records = filter(conf, messages)
       assert_equal(expected, filtered_records)
     end
+    test 'mask field in hash object with exclude' do
+      conf = CONFIG
+      messages = [
+        {:not_masked_field=>"mickey-the-dog", :email=>"mickey-the-dog@zooz.com", :first_name=>"Micky", "excludedField"=>"first_name"}
```

Comment:
`"excludedField"` is not in the `:excludedField` form as the rest of the fields in this hash object